src/main/java/org/intellij/plugin/mdx/highlighting/MdxTemplateHighlighter.kt

```diff
@@ -0,0 +1,29 @@
+package org.intellij.plugin.mdx.highlighting
+
+import com.intellij.lang.ecmascript6.JSXHarmonyFileType
+import com.intellij.lang.javascript.DialectOptionHolder
+import com.intellij.lang.javascript.dialects.ECMA6SyntaxHighlighterFactory
+import com.intellij.openapi.editor.colors.EditorColorsScheme
+import com.intellij.openapi.editor.ex.util.LayerDescriptor
+import com.intellij.openapi.editor.ex.util.LayeredLexerEditorHighlighter
+import com.intellij.openapi.fileTypes.FileType
+import com.intellij.openapi.fileTypes.SyntaxHighlighterFactory
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.vfs.VirtualFile
+import org.intellij.plugin.mdx.lang.parse.MdxTokenTypes
+import org.intellij.plugins.markdown.lang.MarkdownElementType
+import org.intellij.plugins.markdown.lang.MarkdownTokenTypes
+
+class MdxTemplateHighlighter(project: Project?, virtualFile: VirtualFile?, colors: EditorColorsScheme)
```

Comment:
why Template? I believe it was incorrect copy-paste from my side. Let's choose a better name

Reply:
Maybe just MdxHighlighter better?

Reply:
MdxEditorHighlighter